understanding/20/contrast-enhanced.html

```diff
@@ -73,13 +73,19 @@ <h2>Intent of Contrast (Enhanced)</h2>
       </div>
 
       <div class="note">
-         <p>Because authors do not have control over user settings for font smoothing/anti-aliasing, when evaluating this
-            Success Criterion, refer to the foreground and background colors obtained from the user agent, or the underlying
-            markup and stylesheets, rather than the text as presented on screen.</p>
-         <p>Due to anti-aliasing, particularly thin or unusual fonts may be rendered by user agents with a much fainter
-            color than the actual text color defined in the underlying CSS. This can lead to situations where text has
-            a contrast ratio that nominally passes the Success Criterion, but has a much lower contrast in practice.
-            In these cases, best practice would be for authors to choose a font with stronger/thicker lines,
+         <p>User agents which render text will often apply anti-aliasing, font smoothing, and other techniques
+            which can cause some pixels of text to render in fainter colors than those specified by authors.
+            When evaluating this Success Criterion, this is usually not relevant; as long as the applied font,
+            size, and weight ensure that some pixel of each character reaches the author-specified color, evaluators
```

Comment:
but the problem is that you as an author can't always 100% "ensure" anything...

Reply:
normatively, there's a world of difference between saying "yes, there are ways in which authors can increase their chance that antialiasing is likely not to result in things rendering with too low a contrast" (such as choosing a particular font that, in their testing, generally results in strong enough shapes that don't antialias too much, or setting `font-smoothing` in CSS, or starting off with colours that don't just ride the edge of conformance), and "if authors don't use these techniques, then they normatively fail"